README.md

```diff
@@ -2,122 +2,179 @@ energy-blockchain
 ===
 Green Energy Blockchain Project cooperate with NCKU-EE
 
-# API
-## Data Receive.
-Now Support:
+## Getting Started
+
+### Prerequisites
+
+environment
+- python 3.6.8
+- docker 19.03.5
+
+encrypt key files (put in `ami/rsa/`)
+- plat_rsa_public.pem
+- ami_rsa_private.pem
+```
+# Generate AMI Key
+openssl genrsa -out ami_rsa_private.pem 1024
+# 使用 RSA 私鑰產生公鑰
+openssl rsa -in ami_rsa_private.pem -out ami_rsa_public.pem -outform PEM -pubout
+# Generate Platform Key
+openssl genrsa -out plat_rsa_private.pem 4096
+# 使用 RSA 私鑰產生公鑰
+openssl rsa -in plat_rsa_private.pem -out plat_rsa_public.pem -outform PEM -pubout
+```
+
+### Running Development
+
+Installing Packages & Running
+```
+pipenv install
+pipenv shell
+cd ami/
+python app.py
+```
+
+### Running Production
```

Comment:
Discussion:
I think 'docker build instructions' should place above the the Running Production section. I think in first time deployment perspective, putting instructions above it will improve the deployment flow.
- deployer docker login
- deployer try to build image for production (the instructions)
- deployer can run docker
- deployer can rebuild docker image if needed

Reply:
Fix in 26cea38.